Gems/Atom/RHI/DX12/Code/Source/RHI/StreamingImagePool.cpp

```diff
@@ -438,20 +478,41 @@ namespace AZ
             {
                 // The committed image would allocate heap from the entire image
                 // We would only need to update memory usage once at creating time and when resource is shutdown
-
                 D3D12_RESOURCE_ALLOCATION_INFO allocationInfo;
                 GetDevice().GetImageAllocationInfo(request.m_descriptor, allocationInfo);
+
                 auto& memoryAllocatorUsage = GetDeviceHeapMemoryUsage();
-                if (!memoryAllocatorUsage.CanAllocate(allocationInfo.SizeInBytes))
+                
+                bool canAllocate = memoryAllocatorUsage.CanAllocate(allocationInfo.SizeInBytes);
```

Comment:
They are different. The one inside AllocateImageTilesInternal re-evaluate memory needed because the number can be decreased (it can use some released tiles from allocated heaps rather than allocate more heaps)